lib/new_relic/agent/datastores/mongo/metric_translator.rb

```diff
@@ -119,7 +119,7 @@ def self.find_and_remove?(name, payload)
             name == :findandmodify && payload[:selector] && payload[:selector][:remove]
           end
 
-          def self.create_indexes?(name, paylod)
+          def self.create_indexes?(name, payload)
```

Comment:
Scary?

Reply:
It seems we don't use that argument! 

If you wanted to make a further change, you could prefix the name with an underscore to show it is unused, or remove the argument here and where the method is called on line 35.

Underscore example: 
```diff
- def self.create_indexes?(name, payload)
+ def self.create_indexes?(name, _payload)
```

Reply:
I'm willing to add an underscore. Removing an argument is too much for a large spelling fix. (I'd be willing to make an additional PR to do it...).